data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/utils/SinkStopWatch.java

```diff
@@ -0,0 +1,40 @@
+package org.opensearch.dataprepper.plugins.sink.utils;
```

Comment:
This class is not needed and stopwatch methods can be moved to the service class.